testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -204,9 +220,18 @@ def test_set_residues(self):
 
 
 class TestResnames(TestResidueAttr):
-    values = np.array(['ARG', 'LYS', 'VAL', 'POPG'], dtype=np.object)
+    values = np.array(['VAL', 'LYS', 'VAL', 'POPG'], dtype=np.object)
     attrclass = tpattrs.Resnames
 
+    def test_get_named_residue(self):
```

Comment:
I need some more explanation for this. We have a class method that is not supposed to be called by the class itself. How does that work?